challenges/workflow-engine.md

```diff
@@ -1,8 +1,13 @@
 # Workflow Engine
 
-**Allotted Time:** 1 hour
+This assignment utilizes the Sophos Factory platform. You'll build one or more [step modules](https://docs.refactr.it/docs/reference/step-module-reference/) that execute a script in the language of your choice to solve the challenge.
```

Comment:
Could be good to add estimated time for the assignment so that the candidate better understand the expected size/complexity of the solution.

Reply:
Maybe there is a risk that the candidate believes that there is a platform with code (Sophos Factory) needed to be included/installed and imported in the candidate's solution? I'm guessing that the candidate is expected to follow the same concepts used, but not actually use Sophos Factory in the code?!